scripts/linters/js_ts_linter.py

```diff
@@ -59,7 +59,10 @@
 # questions, please talk to @srijanreddy98.
 INJECTABLES_TO_IGNORE = [
     'MockIgnoredService', # This file is required for the js-ts-linter-test.
-    'UpgradedServices' # We don't want this service to be present in the index.
+    'UpgradedServices', # We don't want this service to be present in the index.
+    # This file has location injected in the constructor,
+    # and may cause errors as it alters URL in windows.
+    'TopicEditorRoutingService'
```

Comment:
Move the comments above the files.
```suggestion
    # We don't want this service to be present in the index.
    'UpgradedServices', 
    # This file has location injected in the constructor,
    # and may cause errors as it alters URL in windows.
    'TopicEditorRoutingService'
```

Reply:
Done